tests/geometry-calculation.test.js

```diff
@@ -0,0 +1,120 @@
+const geometry = require('../modules/geometry-calculation')
+const Point = require('../modules/ecs6-class/point')
+const Line = require('../modules/ecs6-class/line')
+
+const point1 = new Point({ x: 1, y: 1 })
+const point2 = new Point({ x: 2, y: 2 })
+const line1 = new Line({ point1, point2, n: 2, slope: 2 })
+
```

Comment:
don't declare parameters in the head of the module, declare each one inside the test